docs/start/install.mdx

```diff
@@ -221,9 +222,8 @@ Additionally, Qiskit only supports the CPython implementation of the Python lang
 
   - Linux ppc64le (distributions compatible with the [manylinux 2014](https://www.python.org/dev/peps/pep-0599/)packaging specification)
   - Linux s390x (distributions compatible with the [manylinux 2014](https://www.python.org/dev/peps/pep-0599/) packaging specification)
-  - macOS ARM64 (10.15 or newer)
   - Linux i686 (distributions compatible with the [manylinux 2014](https://www.python.org/dev/peps/pep-0599/) packaging specification)
-  - Windows 32 bit
+  - Windows 10 32 bit
```

Comment:
Same comment as above: can we do 32-bit hyphenated?

Reply:
```suggestion
  - Windows 10 32-bit
```